egs/ami/s5/cmd.sh

```diff
@@ -12,6 +12,21 @@
 
 export train_cmd="queue.pl --mem 1G"
 export decode_cmd="queue.pl --mem 2G"
+export tfrnnlm_cmd="queue.pl -l hostname=b*"  # this is specific to the CLSP grid
+# we limit to certain machines because TF binaries compiled from C++ take advantage
+# of machine architectures for optimization
+
+export tfrnnlm_py_cmd="queue.pl --gpu 1 CUDA_VISIBLE_DEVICES=\`/home/gkumar/scripts/free-gpu\`"
```

Comment:
OK I have just completely re-written a wrapper that does the job. I will push the commits soon.